colibri/Cargo.toml

```diff
@@ -24,3 +24,13 @@ clap = { version = "4.5.26", features = ["string"] }
 # for globalDB and other sqlite operations
 rusqlite = { version = "0.32.1", features = ["bundled", "bundled-sqlcipher-vendored-openssl"] }
 async-sqlite = {version = "0.4.0" }
+# for cors and network
+tower-http = {version = "0.6.2", features = ["cors", "trace"]}
+http = "1.0"
+# for detailed logs when requests happen
+tracing = "0.1.37"
```

Comment:
so this is yet a different library? https://crates.io/crates/tracing

Is this close to what lighthouse uses? Is it a heavily used solution by many other projects? I am not familiar nor do I have time to research so I need to trust you here.

Just make sure to use something that is not heavy and does not affect performance, does everything we need and is used by many other projects. The last one is important, because if it's something small, niche and new there is a big chance of it just dissapearing down the line

Reply:
This is a library created by tokio and maintained. Is the same one that lighthouse uses https://github.com/sigp/lighthouse/blob/029b4f21047c37d1ffde51c554737b1aa0880f97/Cargo.toml#L194 and I don't expect it to get unmaintained from one day to another.